frontend/src/app/commonComponents/TestResultGuidance/CovidResultGuidance.tsx

```diff
@@ -24,36 +24,20 @@ const CovidResultInfo = ({ result, isPatientApp }: CovidResultInfoProps) => {
             <li>{t("testResult.notes.positive.guidelines.li4")}</li>
             <li>{t("testResult.notes.positive.guidelines.li5")}</li>
           </ul>
-          {isPatientApp ? (
```

Comment:
ooo nice cleanup! I didn't realize this was the same content when I initially refactored this! 😅